services/web/server/src/simcore_service_webserver/workspaces/_groups_db.py

```diff
@@ -62,12 +68,14 @@ async def create_workspace_group(
             )
             .returning(literal_column("*"))
         )
-        row = await result.first()
+        row = result.first()
```

Comment:
And here. I would expect this one to be await maybe the `execute` above should be a `stream`?